config/routes.rb

```diff
@@ -0,0 +1,6 @@
+Rails.application.routes.draw do
+
+  resources :posts, only: [:index, :show, :new, :create, :edit, :update, :destroy]
```

Comment:
if you are using all 7 actions you don't need to say `only:` and could simply say `resources :posts`